src/pages/index.md

```diff
@@ -1,85 +1,18 @@
 ---
-title: Overview - Adobe Analytics
-description: This is the overview page of Adobe Analytics
-contributors:
-  - https://github.com/icaraps 
+title: Commerce Extensibility
+description: An overview of the Adobe Commerce extensibility platform.
 ---
 
-<Hero slots="heading, text"/>
+<Hero slots="image, heading, text"/>
 
-# Adobe Analytics API
+![Commerce Extensibility](_images/home-bg.jpeg)
 
-Adobe Product API offers limitless ways to integrate your most important customer data into key business processes. Adobe Product API offer limitless ways.
+# Adobe Commerce Extensibility
 
-<Resources slots="heading, links"/>
+Learn how to use the Adobe infrastructure to create extensions for Adobe Commerce and Magento Open Source.
 
-#### Resources
+*  [App Builder](https://developer.adobe.com/app-builder/docs/overview/) is a complete framework that enables enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure.
```

Comment:
```suggestion
*  [App Builder](https://developer.adobe.com/app-builder/docs/overview/) is a complete framework that allows enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure.
```

Reply:
I'm going to keep this one as is. We're empowering developers, not giving them permission.